frontend/models/product.ts

```diff
@@ -425,29 +430,16 @@ export type ReviewAuthor = {
 };
 
 export async function fetchProductReviews(
-   apiOrigin: string,
+   apiClient: IFixitAPIClient,
    productId: string
 ): Promise<ProductReviewData | null> {
-   const response = await fetch(
+   const response = await apiClient.get(
```

Comment:
Why don't we just instantiate the apiclient here rather than passing it in?

Reply:
I wonder why we weren't using the ifixit api client here before. Weird.

Reply:
Any idea what the 401 handling was about? I didn't think product reviews would return that.

Reply:
Likely the author didn't know it existed. I made #949

Totally can, same result. I left it out to stick to the pattern (the shopify sdk is also created at the top level) and this encourages re-using the client if we make more than one api call later.

Reply:
I think it was vestigal. Perhaps this code was copied from the user-api request code?

Definitely doesn't return 401:
https://github.com/iFixit/ifixit/blob/master/Guide/UI/api/2.0/reviews.php#L39-L53